bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueSensorHandler.java

```diff
@@ -157,9 +158,44 @@ public void dispose() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no commands
-    }
+        handleCommand(channelUID.getId(), command);
+    }    
+    
+    public void handleCommand(String channel, Command command) {
+        //updateSensorState
+        FullSensor sensor = getSensor();
+        if (  sensor == null) {
+            logger.debug("hue sensor not known on bridge. Cannot handle command.");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.conf-error-wrong-light-id");
+            return;
+        }
+        
+        HueClient hueBridge = getHueClient();
+        if (hueBridge == null) {
+            logger.warn("hue bridge handler not found. Cannot handle command without bridge.");
+            return;
+        }
 
+        StateUpdate sensorState = new StateUpdate();
+        switch (channel)
+        {
```

Comment:
Please use the ESH formatter (automatically on save in Eclipse if you use openHAB IDE setup).

Reply:
OK, I will modify that in next commit